pluralize.js

```diff
@@ -49,7 +49,10 @@
     // Tokens are an exact match.
     if (word === token) return token;
 
-    // Upper cased words. E.g. "HELLO".
+    // Lower cased words. E.g. "hello".
+    if (word === word.toLowerCase()) return token.toLowerCase();
```

Comment:
Looks good. Is it possible to remove the title case check now though and fallback to it by default if neither of these cases match? No need to check if that should be the default now anyhow.

Reply:
I think if we did that things like soundFX, oDonald etc would fail